test/Libraries/RevitNodesTests/Elements/Views/LegendTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using Revit.Elements;
+using RevitServices.Persistence;
+using RevitTestServices;
+using RTF.Framework;
+
+namespace RevitNodesTests.Elements.Views
+{
+    [TestFixture]
+    public class LegendTests : RevitNodeTestBase
+    {
+        [Test, TestModel(@".\View\Legend.rvt")]
+        public void GetLegend()
+        {
+            var legend = new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                .OfClass(typeof(Autodesk.Revit.DB.View))
+                .Cast<Autodesk.Revit.DB.View>()
+                .FirstOrDefault(x => x.ViewType == Autodesk.Revit.DB.ViewType.Legend);
+            Assert.NotNull(legend);
+
+            var wrappedLegend = legend.ToDSType(true);
+            Assert.Equals(wrappedLegend.GetType(), typeof(Revit.Elements.Views.Legend));
```

Comment:
This Assert.Equals throws exception “Assert.Equals should not be used for assertions”.
It would be successful that use Assert.AreEqual instead.